templates/procuration/landing.html.twig

```diff
@@ -14,39 +14,41 @@
                     <a href="{{ path('homepage') }}" class="text--logo link--no-decor text--white">En Marche !</a>
                 </div>
                 <div class="procuration__header--inner l__wrapper--slim">
-                    <h1 class="text--larger">
-                        Chaque vote compte.
-                    </h1>
-                    <h2 class="text--medium b__nudge--top l__hide--on-mobile b__nudge--bottom-small">
-                        Les élections législatives ont lieu les 11 et 18 juin 2017 en France (3 et 17 juin pour les Français de l'étranger du continent Américain et 4 et 18 juin pour les autres Français de l'étranger).
-                    </h2>
-                    <div class="text--body">Si vous ne votez pas en France métropolitaine, <a href="https://www.service-public.fr/particuliers/actualites/A10598" class="link--white">renseignez-vous sur les dates</a>.</div>
+                    {% if next_election %}
+                        {{ next_election.introduction|raw }}
+                    {% else %}
+                        No coming election, TODO.
```

Comment:
The `TODO` text is needed ?

Reply:
Not more than the English ;), but I'm making small PRs targeting the `Feature/procuration` branch, to ease review and merge quickly. The branch is still WIP and I'm waiting for some content from Marine.